modules/gltf/doc_classes/GLTFAccessor.xml

```diff
@@ -58,4 +58,27 @@
 			The GLTF accessor type as an enum. Use [member accessor_type] instead.
 		</member>
 	</members>
+	<constants>
+		<constant name="TYPE_SCALAR" value="0" enum="GLTFAccessorType">
+			Accessor type "SCALAR", enumerated as 0. For the glTF object model, this can be used to map to a single float, int, or bool value, or a float array.
```

Comment:
Is it really necessary to write the value of the enum in the description when it's already automatically part of the docs?

Reply:
I suppose not.